sinks/librato.py

```diff
@@ -288,13 +285,16 @@ def add_measure(self, key, value, time):
             k = name
 
         if k not in self.measurements:
-            self.measurements[k] = {
-                'name': name,
-                'tags' : tags,
-                'time' : ts
-            }
+            self.measurements[k] = []
```

Comment:
I fixed this by making tags an `OrderedDict` and then incorporating that into "k" instead of/in addition to "source".